### PR TITLE
Worn wigs no longer show up on examine

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Wigs no longer show up if you examine a person wearing one

## Why It's Good For The Game
Requested, makes the lives of bald people a tad less shameful

## Changelog
:cl: Ryll/Shaps
tweak: Wigs no longer appear on the examine lists of people wearing them
/:cl:

### CODE REVIEW

**Comment by Ryll-Ryll** [comment]
FAQ:

Q: It'd be funny if you could spot a wig if you're close/it's low quality/if XYZ
A: That's not a question, and if you have a funny idea for when wigs should/shouldn't be visible, feel free to make a PR with it

File: code/__DEFINES/inventory.dm [hunk 1]
```diff
@@ -73,13 +73,14 @@
...
 #define HIDESHOES		(1<<3)
-#define HIDEMASK		(1<<4)	//these last six are only used in masks and headgear.
+#define HIDEMASK		(1<<4)	//these next seven are only used in masks and headgear.
 #define HIDEEARS		(1<<5)	// (ears means headsets and such)
 #define HIDEEYES		(1<<6)	// Whether eyes and glasses are hidden
 #define HIDEFACE		(1<<7)	// Whether we appear as unknown.
 #define HIDEHAIR		(1<<8)
 #define HIDEFACIALHAIR	(1<<9)
 #define HIDENECK		(1<<10)
+#define HIDEHEADGEAR	(1<<11) // for wigs, only obscures the headgear
```
Comment (Jared-Fogle) [inline code/__DEFINES/inventory.dm:83]:
```suggestion
#define HIDEHEADGEAR	(1<<11) /// for wigs, only obscures the headgear
```

**Comment by uomo91** [comment]
> FAQ:
> 
> Q: It'd be funny if you could spot a wig if you're close/it's low quality/if XYZ
> A: That's not a question, and if you have a funny idea for when wigs should/shouldn't be visible, feel free to make a PR with it

This is a great opportunity for somebody to code in a toupée

**Comment by Donkie** [comment]
Collides with my #53241 so we'll have to coordinate that

**Comment by Ryll-Ryll** [comment]
@Donkie i can wait for you to get merged first, this isn't a super high prio, i'll update mine per your changes once you get merged

**Comment by cacogen** [comment]
I like disguise

**Comment by Donkie** [comment]
@Ryll-Ryll My PR is merged now so rebase this

File: code/modules/mob/living/carbon/human/examine.dm [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 		. += "[t_He] [t_is] wearing [w_uniform.get_examine_string(user)][accessory_msg]."
 	//head
-	if(head)
+	if(head && !(ITEM_SLOT_HEAD in obscured))
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/human/examine.dm:32]:
```suggestion
	if(head && !(ITEM_SLOT_HEAD & obscured))
```
`obscured` is not a list, it's a bitflag variable, right?

**Pull request merged**